server/graphql-schema.js

```diff
@@ -107,6 +107,7 @@ const graphqlSchema = gql`
         # isLatest: Boolean!
         updatedAt: Timestamp!
         exampleUrl: String!
+        directory: String!
```

Comment:
Well would there ever be any need for the application to know the actual ID of the `TestPlanVersion` record? How to definitely know a previously reported on version has been been outdated comes to mind.

Reply:
I'm not sure I understand that! There is an ID for TestPlanVersions, right?

Reply:
Apologies, I just noticed the distinction you were making between 'Test Plan' and 'Test Plan Version' here.

Well with that being the case, I suppose it's just for us to finalize the following query:

```graphql
testPlans {
  latestTestPlanVersion {
    id
    title
  }
}
```
Also with the inclusion of that `id` field as the `directory` like you've mentioned. 

What I've experienced with it to be clear is that it outputs the information for just a single test plan ('checkbox') all the time. With that complete, I suppose it would clear this up in the intended way